cloud/blockstore/libs/rdma/impl/server.cpp

```diff
@@ -1276,6 +1277,20 @@ class TCompletionPoller final
         }
     }
 
+    void DisconnectAllSessions()
+    {
+        auto sessions = Sessions.Get();
+
+        for (const auto& session: *sessions) {
+            session->Flush();
+        }
+
+        while (sessions->size()) {
```

Comment:
Comment explaining that we are waiting for a main loop to close flushed or timed out sessions would be nice